test/helpers/cmake/builddirectoryhelper.ts

```diff
@@ -0,0 +1,20 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import * as rimraf from 'rimraf';
+
+export class BuildDirectoryHelper {
+
+  private readonly location: string;
+
+  public constructor(location: string) { this.location = location; }
```

Comment:
A shorthand:

```typescript
public constructor(private readonly location: string) {}
```

will declare and set the property `location` to the first argument passed to the constructor. No need to declare the field or do an explicit assignment. It's used in a few places in the extension.

Reply:
Done